en/02_Developer_Guides/01_Templates/01_Syntax.md

```diff
@@ -69,32 +68,33 @@ This will result in trying to fetch a `Title` property from the data being displ
 Variables can be chained together, and include arguments.
 
 ```ss
-$Foo(param)
+$Foo("param")
 $Foo.Bar
 ```
 
 These variables will call a method / field on the object and insert the returned value as a string into the template.
 
+Arguments don't have to be literals - you can pass template variables as arguments as well.
+
 > [!WARNING]
-> If you wish to pass arguments to getter functions, you must use the full method name. e.g. `$Thing` will try to access `Thing` as a property, which will ultimately result in `getThing()` being called with no arguments To pass arguments you must use `$getThing('param')`.
->
-> Also, arguments must be literals, and cannot be other template variables (`$getThing($variable)` will pass the literal string `'$variable'` to the `getThing()` method).
+> Arguments cannot be literal arrays (e.g `$myMethod(["val1", "val2"])`). The behaviour for this is currently undefined.
```

Comment:
This was mostly saying "you can't pass template variables as arguments" which as noted above is no longer the case. It's just array literals you can't do now.